packages/schema/index.ts

```diff
@@ -0,0 +1,47 @@
+export default {
+  'link': {
+    type: 'inline',
+    attributes: [
+      'destination',
```

Comment:
I'd err on the side of `uri` since that's a well-understood term and valid everywhere. Possibly `url` if not `uri`

Reply:
`url` probably a better choice that `uri` for ease-of use